smallrye-reactive-messaging-kafka/pom.xml

```diff
@@ -36,6 +36,22 @@
       <version>${kafka.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>io.opentracing</groupId>
+      <artifactId>opentracing-api</artifactId>
+      <version>0.31.0</version>
+    </dependency>
+    <dependency>
+      <groupId>io.opentracing</groupId>
+      <artifactId>opentracing-util</artifactId>
+      <version>0.31.0</version>
+    </dependency>
+<!--    <dependency>-->
```

Comment:
Any reason the `opentracing-kafka-streams` artifact didn't work? Should probably remove it too

Reply:
This is a good question. In the last commit I have changed to use `opentracing-kafka-client`. The tracing provides the same visibility, with one difference - in the Kafka message is stored immutable `SpanContext` instead of `Span`. 

I think it's better bc provided span was anyways finished so users could not add any data to ti.